rust/template/Cargo.toml

```diff
@@ -17,10 +17,11 @@ path = "./cmd_parser"
 path = "./ovsdb"
 
 [dependencies]
+fallible-iterator = "0.2"
```

Comment:
Are we still using `FallibleIterator`?

Reply:
I'll change to `FallibleIterator` once the API comes into shape, trying to keep it simple for now.

Reply:
I'm not sure it's needed given the current design philosophy to only push data that is ready in the producer.

Reply:
I see!